presto-docs/src/main/sphinx/ecosystem/list.rst

```diff
@@ -0,0 +1,34 @@
+=====================
+Ecosystem
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+Here are some projects that support Presto:
+
+
+**Admin**
+
+* `presto-admin <https://github.com/prestodb/presto-admin>`_ : A tool to install, configure and manage Presto installations.
```

Comment:
This repo does not appear to have been updated since 2017. Is this the best admin tool that can be linked to now? Does this tool work with current Presto? If not, maybe consider finding a more current tool, or deleting this item.

Reply:
removed this repo